addons/mail/controllers/main.py

```diff
@@ -119,6 +119,7 @@ def read_followers(self, follower_ids, res_model):
             followers.append({
                 'id': follower.id,
                 'name': follower.partner_id.name or follower.channel_id.name,
+                'display_name': follower.partner_id.display_name or follower.channel_id.name,
```

Comment:
Maybe also use `display_name` from channel for consistency?

Reply:
Right, changed to `display_name`.